sphinxcontrib/openapi/openapi20.py

```diff
@@ -65,9 +73,98 @@ def _httpresource(endpoint, method, properties):
             for line in header['description'].splitlines():
                 yield '{indent}{indent}{line}'.format(**locals())
 
+    for status, response in responses.items():
+        if not is_2xx_response(status):
```

Comment:
Why non-2xx are ignored? All other responses pretty often have the schema too, so why ignore them?